packages/shared/routes/onboarding/views/recovery/views/FileImportView.svelte

```diff
@@ -44,16 +48,12 @@
             (event as DragEvent)?.dataTransfer?.files?.[0] ?? (event?.target as HTMLInputElement)?.files?.[0] ?? null
```

Comment:
In this function file should be renamed to `fileWithPath` to prevent confusion with file (or the other `file` should be `fileBuffer`). Then the global variable (`file` / `fileBuffer`) can be just of type `ArrayBuffer | string`.